plugins/woocommerce/client/legacy/js/frontend/cart.js

```diff
@@ -248,7 +248,10 @@ jQuery( function ( $ ) {
 				data: data,
 				dataType: 'html',
 				success: function ( response ) {
+					var target_id = $( event.currentTarget ).attr( 'id' );
+
 					update_cart_totals_div( response );
+					$( '.cart_totals' ).find( '#' + target_id ).focus();
```

Comment:
Since we now moving away from jQuery code, can you please write native JS code instead for this.

Reply:
Thanks for the feedback, @naman03malhotra .
Please review it: [Refactor the code to use native JS instead of jQuery](https://github.com/woocommerce/woocommerce/pull/49360/commits/c8c7f8b6928b8a6e51e70e0c2275128e253037b9)